ehrapy/anndata/_constants.py

```diff
@@ -2,7 +2,13 @@
 # -----------------------
 # The column name and used values in adata.var for column types.
 
-EHRAPY_TYPE_KEY = "ehrapy_column_type"
+EHRAPY_TYPE_KEY = "ehrapy_column_type"  # TODO: Change to ENCODING_TYPE_KEY
```

Comment:
is ENCODING_TYPE_KEY still intended?
it is not for encoded types only, right?

Reply:
Ah, I see below there is new types to be coming

Reply:
Yeah, we decided to improve this in two steps. This PR alone is already a lot of work and @Lilly-May will revisit it again later.